### PR TITLE
Add support for files

### PR DESCRIPTION
**Pull request by SferaDev**
Meta ticket: https://github.com/xataio/frontend-next/issues/2144

### CODE REVIEW

File: packages/client/src/api/fetcher.ts [hunk 1]
```diff
@@ -161,6 +161,8 @@ export async function fetch<
         Authorization: `Bearer ${apiKey}`
       };
 
+      console.log('body', body);
+
```
Comment (SferaDev) [inline packages/client/src/api/fetcher.ts:165]:
```suggestion
```

File: packages/client/src/schema/repository.ts [hunk 2]
```diff
@@ -1978,9 +2015,11 @@ export const initObject = <T>(
   const { columns } = schemaTables.find(({ name }) => name === table) ?? {};
   if (!columns) console.error(`Table ${table} not found in schema`);
 
+  const columnsToInit = compact(selectedColumns?.map((column) => column.split('.')[0]) ?? []);
+
```
Comment (SferaDev) [inline packages/client/src/schema/repository.ts:2019]:
```suggestion
```

File: packages/client/src/schema/repository.ts [hunk 3]
```diff
@@ -1978,9 +2015,11 @@ export const initObject = <T>(
...
   for (const column of columns ?? []) {
     // Ignore columns not selected
-    if (!isValidColumn(selectedColumns, column)) continue;
+    if (!isValidColumn(columnsToInit, column)) continue;
```
Comment (SferaDev) [inline packages/client/src/schema/repository.ts:2022]:
```suggestion
    if (!isValidColumn(selectedColumns, column)) continue;
```

**Comment by richardgill** [comment]
The code LGTM. 

Will someone be dog fooding this to check it out?

**Review by richardgill — APPROVED** [review]

**Comment by SferaDev** [comment]
Yeah, we have ~15 people testing it out from the EAP

**Pull request merged**